drivers/sensor/max17055/max17055.c

```diff
@@ -72,6 +94,29 @@ static int capacity_to_ma(unsigned int rsense_mohms, int16_t val)
 	return rem;
 }
 
+/**
+ * @brief Convert capacity in milliamphours to MAX17055 units
+ *
+ * @param rsense_mohms Value of Rsense in milliohms
+ * @param val_mha Value in milliamphours to convert
+ * @return corresponding value in MAX17055 units, ready to write to a register
+ */
+static int capacity_to_max17055(unsigned int rsense_mohms, uint16_t val_mha)
+{
+	return (val_mha * rsense_mohms) / 5.0;
```

Comment:
Why 5.0?

This seems to be using float and then converting back to int, right?

Also, drop the brackets?

Reply:
Sorry - no need for `5.0` to be a float.

The motivation for the brackets is to ensure that `val_mha` is multiplied by `rsense_mohms` before the division operation. I believe C follows BODMAS?

`rsense_mhoms` can reasonably be small - in my application it's `10`, but a value of `4` or lower would be reasonable. Without brackets, i believe we'd end up multiplying `val_mha` by `0` in this case?

Reply:
Ignore me. This is garbage. Sorry - fix incoming.